CHANGELOG.md

```diff
@@ -66,6 +66,8 @@ Not yet released.
   - [#3504](https://github.com/bpftrace/bpftrace/pull/3504/)
 - Introduce builtin to access percpu kernel data
   - [#3596](https://github.com/bpftrace/bpftrace/pull/3596/)
+- Pointers may now be used in if conditions, tenary conditions and as operands in logical AND and OR expressions
```

Comment:
This change is going into master and not the release branch (at least for now) so let's move this into the un-released section.

Reply:
Ah, moved. Hopefully this is the right spot now.